### DISCUSSION TITLE
Opening Konfetti Discussions

### DISCUSSION THREAD

**Discussion by DanielMartinus** [DanielMartinus/Konfetti#229] [General]
## Welcome everyone! 👋
  Discussions have now been opened up for this repository. 🥳 

_Soon I'll remove the telegram link from main page and discussions will be the new place to reach out._

### What is this used for?
  * Ask for help or ask technical questions to the maintainers and contributors
  * Share ideas on how to improve or extend Konfetti
  * Remember to stay inclusive. Everyone is welcome to share ideas, start discussions etc. 
  * Found a bug or an issue? We still use the [issue section](https://github.com/DanielMartinus/Konfetti/issues) for this

**Comment by dcon01** [comment]
Is it possible to have text behind the confetti?

**Reply by Buggyy** [reply]
When triggering the confetti, you can show a Text composable?

**Comment by Buggyy** [comment]
I can't seem to wrap my head around the compose sample, can you implement it without states?
